qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -0,0 +1,40 @@
+from qcodes.instrument.visa import VisaInstrument
+from qcodes.utils.validators import Numbers
+
+
+class GS200(VisaInstrument):
+    """
+    This is the qcodes driver for the Yokogawa GS200 voltage and current source
+
+
+    TODO:(nataliejpg)
+    - add current functionality (mode settings)
+    """
+
+    def __init__(self, name, address, reset=False, **kwargs):
+        """
+        Args:
```

Comment:
🎠   Class docstring is split, and should all be in under the class and not on the init function.
Reset  attr needs to be described. Also, is it used ? 

See http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for an example.